synapse_invite_checker/invite_checker.py

```diff
@@ -167,21 +171,38 @@ def __init__(self, config: InviteCheckerConfig, api: ModuleApi):
         ) as db_conn:
             self.store = InviteCheckerStore(api._store.db_pool, db_conn, api._hs)
 
-        InfoResource(self.config, self.__version__).register(self.resource)
-        ContactsResource(self.api, self.store, self.config).register(self.resource)
-        ContactResource(self.api, self.store, self.config).register(self.resource)
+        # This first InfoResource registers the contact management metadata. It needs to
+        # be on the v1 endpoint for backwards compatibility.
+        InfoResource(self.config, self._TMCM_version, self.config.api_prefix).register(
```

Comment:
Please split the different APIs cleanly into separate files and register them as separate resources. They do different things and one of them is new and the other one we want to delete in the near future. Mixing them makes this significantly harder, especially if you order them here:

AInfo
BInfo
AContacts
AContact

Reply:
You got it

Reply:
Sorted out in [216068e](https://github.com/famedly/synapse-invite-checker/pull/33/commits/216068ecdabc32a7edc50ac22a5a6119a1eda6a1)